third_party/2and3/Crypto/Random/random.pyi

```diff
@@ -1,18 +1,18 @@
-from typing import Any, List, Optional, Sequence, TypeVar
+from typing import Any, List, Optional, Union, Sequence, TypeVar
 
 _T = TypeVar('_T')
 
 class StrongRandom:
     def __init__(self, rng: Optional[Any] = ..., randfunc: Optional[Any] = ...) -> None: ...
     def getrandbits(self, k: int) -> int: ...
-    def randrange(self, *args: int) -> int: ...
+    def randrange(self, start: int, stop: Union[int, None] = ..., step: int = ...) -> int: ...
```

Comment:
Looking at the implementation, there is an `isinstance` check that won't allow `None` for `stop`.

Reply:
You're right, it should be `Optional`.

Reply:
`Optional` is the same as `Union[..., None]`. I think it should just be `int`.

Reply:
It's an error if you explicitly pass `None` (so I misinterpreted why you wanted the above change, but think `Optional` is better here anyway), but one, two or three `int` arguments are all correct and supported.

Reply:
`Optional` is unfortunately misnamed. To mark an argument as optional use `= ...` (like you did). `Optional[X]` is only an alias for `Union[None, X]`, so annotating `stop` with `Optional[int]` means that it will accept both `int` and `None` values.

Reply:
Oh! Gotcha. Thanks.